Agents/utils/chemistry_and_robots/chemistry_and_robots/kg_operations/sparql_client.py

```diff
@@ -26,8 +26,34 @@
 
 
 class ChemistryAndRobotsSparqlClient(PySparqlClient):
+    def upload_ontology_tbox(self, tbox_namespace, alternative_url=None):
+        try:
+            query = f'SELECT * WHERE {{ <{tbox_namespace}> <{OWL_VERSION}> ?v. }}'
+            res = self.performQuery(query)
+        except Exception as ex:
+            logger.error("Unable to retrieve TBox version from KG.")
+            raise Exception("Unable to retrieve TBox version from KG.") from ex
+
+        # Upload TBox if not already instantiated
+        if not res:
+            temp_fp = '_tmp.owl'
+            url_to_upload = TBOX_DICT.get(tbox_namespace) if not alternative_url else alternative_url
+            try:
+                try:
+                    content = requests.get(url_to_upload)
```

Comment:
Rather than downloading the OWL files from GitHub, it would be much preferable to generate them from their CSV sources using the TBox generator.

Reply:
Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/677/commits/8ec3e86bd253891c6aaf87ac5d07c44ea45eab00, https://github.com/cambridge-cares/TheWorldAvatar/pull/677/commits/d43b6c3c9c15e4a152d77b2651051b4e077ef2b0, https://github.com/cambridge-cares/TheWorldAvatar/pull/677/commits/f1495265c74c670e2ce9e35e826ee4d0bc09d4a1, https://github.com/cambridge-cares/TheWorldAvatar/pull/677/commits/3334f425366743d94fa9e3a342a13369e4e2a9b8. 

Made the test release [`chemistry_and_robots==1.6.0.a0`](https://test.pypi.org/project/chemistry-and-robots/1.6.0a0/) and it works as expected. The proper release will be made as part of PR for https://github.com/cambridge-cares/TheWorldAvatar/tree/210-dev-catalyst-selection.